src/main/java/com/denizenscript/clientizen/scripts/containers/gui/elements/BoxElement.java

```diff
@@ -0,0 +1,79 @@
+package com.denizenscript.clientizen.scripts.containers.gui.elements;
+
+import com.denizenscript.clientizen.scripts.containers.gui.GuiScriptContainer;
+import com.denizenscript.denizencore.tags.TagContext;
+import com.denizenscript.denizencore.utilities.YamlConfiguration;
+import com.denizenscript.denizencore.utilities.debugging.Debug;
+import com.denizenscript.denizencore.utilities.text.StringHolder;
+import io.github.cottonmc.cotton.gui.widget.WBox;
+import io.github.cottonmc.cotton.gui.widget.WWidget;
+import io.github.cottonmc.cotton.gui.widget.data.Axis;
+import io.github.cottonmc.cotton.gui.widget.data.HorizontalAlignment;
+import io.github.cottonmc.cotton.gui.widget.data.VerticalAlignment;
+
+import static com.denizenscript.clientizen.scripts.containers.gui.GuiScriptContainer.applyInsets;
+import static com.denizenscript.clientizen.scripts.containers.gui.GuiScriptContainer.getSubPath;
+import static com.denizenscript.clientizen.scripts.containers.gui.GuiScriptContainer.getTaggedEnum;
+import static com.denizenscript.clientizen.scripts.containers.gui.GuiScriptContainer.getTaggedInt;
+
+public class BoxElement implements GuiScriptContainer.GuiElementParser {
+
+    // <--[language]
+    // @name Box GUI Element
+    // @group GUI System
+    // @description
+    // Boxes are a type of panel that that sort the elements they contain evenly along an axis; they have a UI type of "box".
```

Comment:
Perhaps `BoxPanel` / `box_panel` to be consistent with grid panel

Reply:
Fair, renamed